src/disclosures/css/disclosures.less

```diff
@@ -243,21 +243,47 @@
   &_content {
 
     .respond-to-min(@bp-sm-min, {
-      .grid_column(9);
+      .grid_column(6);
     });
+  }
 
-    .respond-to-min(@bp-lg-min, {
-      .grid_column(7);
-    });
+  &_subheading {
+    .heading-4();
+  }
 
-    .respond-to-min(@bp-xl-min, {
+  &_about-wrapper {
```

Comment:
Niq, your CSS still doesn't address the spacing weirdness Candice and I saw in the space between columns. I'll add your suggestion, but still look at that bug in a future PR.